configure.ac

```diff
@@ -106,6 +106,12 @@ XMP_TRY_COMPILE(whether compiler understands -Wall,
   int main(void){return 0;}],
   CFLAGS="${CFLAGS} -Wall")
 
+XMP_TRY_COMPILE(whether compiler understands -Wextra,
+  ac_cv_c_flag_w_extra,
+  -Wextra -Wno-unused-parameter -Wno-sign-compare,[
+  int main(void){return 0;}],
+  CFLAGS="${CFLAGS} -Wextra -Wno-unused-parameter -Wno-sign-compare")
```

Comment:
saving space and extra lines? OK :)

Reply:
The idea was more like if the annoying warnings can't be turned off, just don't add -Wextra at all. `-Wno-sign-compare` is temporary, really—the problem is when I *did* go through looking at places the warning crops up, there are uses where the unsigned side can be pretty arbitrarily large and the signed side intentionally uses negatives to indicate failure states. Untangling them all will probably have to be done gradually.

As for `-Wno-unused-parameter`, it spams out the build process for things that aren't really problems. *Maybe* some of them could be fixed, but most of them are things like `start` which is provided to the format load functions (and only really matters for loaders that can be invoked by other loaders i.e. UMX -> MOD/S3M/XM/IT).

Reply:
I meant checking individual switches, well, individually - but they being not activated without -Wextra, all is good